torch/csrc/jit/codegen/cuda/interface.cpp

```diff
@@ -26,6 +26,32 @@ namespace jit {
 namespace fuser {
 namespace cuda {
 
+#ifdef USE_CUDA
+class LoadingNvfuserLibrary {
+ public:
+  LoadingNvfuserLibrary() {
+    static c10::once_flag load_nvfuser_lib_flag;
+    c10::call_once(load_nvfuser_lib_flag, [&] {
```

Comment:
nit: do we need c10::call_once if this constructor is being called statically?

Reply:
@ngimel there's a bug in std::call_once https://github.com/pytorch/pytorch/pull/78051